src/main/java/frc/robot/commands/grabber/AidenPlaceCommand.java

```diff
@@ -23,7 +23,8 @@ public AidenPlaceCommand(RollerSubsystem rollerSubsystem){
     @Override
     public void initialize() {
         System.out.println("Roller to place");
-    } 
+        timer.start();
+    }
 
     @Override
     public void execute() {
```

Comment:
i cant comment on the isFinished() method but it seems problematic to assume we're done dropping as soon as the limit switch is not pressed anymore